src/vm/builtins/builtin_runner/range_check.zig

```diff
@@ -43,26 +61,188 @@ pub const RangeCheckBuiltinRunner = struct {
         n_parts: u32,
         included: bool,
     ) Self {
+        var bound = Felt252.one().overflowing_shl(16 * n_parts);
+        var _bound: ?Felt252 = if (n_parts != 0 and bound.zero()) null else Felt252.new(bound);
```

Comment:
`bound` and `_bound` could be declared as `const` instead of `var` because they are not modified afterwards. Maybe if this still is readable, for a more compact format, you can directly integrate those lines into the return statement without any variable declaration for a more compact format.

Reply:
Will switch to const that makes sense.

As for the implementation the rust version is nearly identical.
 https://github.com/lambdaclass/cairo-vm/blob/0d41042b906258c28531797045d012a30b036ab1/vm/src/vm/runners/builtin_runner/range_check.rs#L48
I'm not sure that it would be readable if declared in the return as bound would be called twice in the assignment of _bound. It would be very long and personally think less readable.

Reply:
Agreed